subprojects/soak/src/integTest/groovy/org/gradle/vfs/VirtualFileSystemRetentionSoakTest.groovy

```diff
@@ -66,28 +65,44 @@ class VirtualFileSystemRetentionSoakTest extends DaemonIntegrationSpec implement
     }
 
     def "file watching works with multiple builds on the same daemon"() {
+        def numberOfChangesBetweenBuilds = MAX_FILE_CHANGES_WITHOUT_OVERFLOW
+
         when:
         succeeds("assemble")
         def daemon = daemons.daemon
         then:
         daemon.assertIdle()
 
         expect:
-        NUMBER_OF_REPETITIONS.times { iteration ->
-            changeAllSourceFiles(iteration)
+        boolean overflowDetected = false
+        50.times { iteration ->
+            // when:
+            changeSourceFiles(iteration, numberOfChangesBetweenBuilds)
```

Comment:
This will now change the same `numberOfChangesBetweenBuilds` files every build. I guess that's okay, right?

Reply:
Yeah, I think that shouldn't make a difference.

Reply:
Yeah, okay, let's not do it then.

Reply:
👍